src/session.rs

```diff
@@ -7,7 +7,7 @@ use crypto::aes::KeySize;
 use crypto::aes_gcm::AesGcm;
 use crypto::chacha20poly1305::ChaCha20Poly1305;
 use crypto::scrypt::{scrypt, ScryptParams};
-use ring::rand::SystemRandom;
+use ring::rand::{SecureRandom, SystemRandom};
```

Comment:
`SystemRandom` implements the `SecureRandom` trait. Previously there was a `SystemRandom:fill()`  in addition to the `fill()` from `SecureRandom`. However, now that extra `SystemRandom:fill()` is gone, so you have to call `fill()` through the trait, and that requires the trait to be used.